doc/semicons.xml

```diff
@@ -299,3 +299,32 @@ true]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="SemilatticeFromDigraph">
+  <ManSection>
+    <Func Name="SemilatticeFromDigraph" Arg="digraph"/>
+    <Returns>
+      A semilattice of partial perms <A>n</A>.
```

Comment:
What is the point of the `<A>n</A>` here?

Reply:
Ooops, I have no idea. Probably something I forgot to deleted after copying another manual entry... I will get rid of it.